### PR TITLE
Swift Package Manager 

### PR DESCRIPTION
**Pull request by danramteke**

### CODE REVIEW

**Comment by tylerhall** [comment]
Hey, @danramteke. Thanks for submitting this. I appreciate it. A few questions.

1. Could you give me a quick (short, nothing fancy) summary of the bug fixes you included? The `sha256` one is great, but I was curious about a few other changes. I'm going to leave my questions inline in your PR.
2. It also looks like you deleted the `.xcodeproj` file. I know you added support for inclusion via SPM, but is removing the project file a requirement for that?

Thanks!

File: roland/Layouts/LayoutRSS.swift [hunk 1]
```diff
@@ -27,7 +27,7 @@ class LayoutRSS: Layout {
             return nil
         }
 
-        template = Template(templateName: "RSS", website: website)
+        template = Template(templateName: "rss", website: website)
```
Comment (tylerhall) [inline roland/Layouts/LayoutRSS.swift:30]: Was there a reason for lowercasing `rss`?

File: roland/Objects/Page.swift [hunk 2]
```diff
@@ -49,9 +49,9 @@ class Page {
         var foundEndOfFrontMatter = false
         while !foundEndOfFrontMatter {
             if lines.count == 0 {
-                foundEndOfFrontMatter = true
```
Comment (tylerhall) [inline roland/Objects/Page.swift:52]: What was the bug that this fixed?

Comment (danramteke) [reply roland/Objects/Page.swift:52]: this prevented an array index out of bounds error on line 55

File: roland/Layouts/LayoutRSS.swift (hunk 1 above)
Comment (danramteke) [reply roland/Layouts/LayoutRSS.swift:30]: mac is case-insensitve, but Linux isn't.

**Comment by danramteke** [comment]
> 1. Could you give me a quick (short, nothing fancy) summary of the bug fixes you included? The `sha256` one is great, but I was curious about a few other changes. I'm going to leave my questions inline in your PR.

They were all motivated by warnings or errors from `swift build`

> 2. It also looks like you deleted the `.xcodeproj` file. I know you added support for inclusion via SPM, but is removing the project file a requirement for that?

 Of course, feel free to disregard.

**Pull request closed without merging**